tests/generate_code.sh

```diff
@@ -37,6 +37,7 @@ $TEST_NOINCL_FLAGS $TEST_CPP_FLAGS $TEST_CS_FLAGS -I include_test monster_test.f
 ../flatc --rust $TEST_RUST_FLAGS -o monster_test -I include_test monster_test.fbs monsterdata_test.json
 
 ../flatc --python $TEST_BASE_FLAGS -I include_test monster_test.fbs monsterdata_test.json
+../flatc --python --gen-onefile $TEST_BASE_FLAGS -I include_test monster_test.fbs monsterdata_test.json
```

Comment:
this should be added to the generate_code.py script as well.

Reply:
I've updated this PR and add the command to generate_code.py. Please take a look. Thanks!